### DISCUSSION TITLE
Advice for using angular-eslint+prettier for scss files

### DISCUSSION THREAD

**Discussion by trentprynn** [angular-eslint/angular-eslint#871] [Q&A]
Hello, I'm using angular-eslint on a [portfolio website](https://github.com/trentprynn/samueldebartolo.com) to great success so thank you for your work on such a nice project!

Currently I use eslint (angular-eslint) + prettier (eslint-config-prettier + eslint-plugin-prettier) to lint + format the code in my project and it's amazing because I can literally just run `yarn lint` and all of the files are linted and formatted from a single command without any second thought

I would really like to include `.scss` files in this linting + formatting chain but when I add `"src/**/*.scss"` to my the `lintFilePatterns` list in my `angular.json` file I get linting errors that aren't really errors like

```
/samueldebartolo.com/src/styles.scss
2:1  error  Parsing error: Unexpected character '@'

1 problem (1 error, 0 warnings)

Lint errors found in the listed files.
```

I know prettier supports formatting `.scss` files so I was wondering if you had any advice on either of the following
1. Changing something so eslint supports linting the files and they eventually make it to prettier for formatting 
2. Allowing eslint to pass on the files so they make it through to prettier for formatting (what I really care about)

**Comment by MaxEtMoritz** [comment]
I had the same problem and this came up when searching.
I have found a solution involving the npm package [any-eslint-parser](https://www.npmjs.com/package/any-eslint-parser) as parser for scss files.
This is the configuration i ended up with (in eslint's javascript-based config format):
```js
import eslint from '@eslint/js';
import { config, configs as tseslint } from 'typescript-eslint';
import { configs as angular, processInlineTemplates } from 'angular-eslint';
import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
import anyEslintParser from 'any-eslint-parser';

export default config(
  {
    files: ['**/*.ts'],
    extends: [
      eslint.configs.recommended,
      ...tseslint.recommended,
      ...tseslint.stylistic,
      ...angular.tsRecommended,
      eslintPluginPrettierRecommended
    ],
    processor: processInlineTemplates,
    rules: {
     ...
    }
  },
  {
    files: ['**/*.html'],
    extends: [...angular.templateRecommended, ...angular.templateAccessibility],
    rules: {}
  },
  {
    files: ['**/*.scss'],
    extends: [eslintPluginPrettierRecommended],
    languageOptions: { parser: anyEslintParser }
  }
);
```
